game-logic.js

```diff
@@ -0,0 +1,367 @@
+  var str = ``;
+  for (let i = 0; i < ordering.length; i += 1) {
+    str += `${ordering[i]}`;
+  }
+
+  var concatVal = Number(str);
+  return Number(concatVal);
+};
+
+// This function should be called when all player had chosen their ordering.
+const decideRoundWinner = function () {
+  var max = 0;
+  var playerWinners = [];
+  for (
+    let playerIndex = 0;
+    playerIndex < playerOrderingChoice.length;
+    playerIndex++
+  ) {
+    var val = concatenatedVal(playerOrderingChoice[playerIndex]);
+    if (val > max) {
+      playerWinners = [];
+      max = val;
+      playerWinners.push(playerIndex);
+    } else if (val === max) {
+      playerWinners.push(playerIndex);
+    }
+  }
+
+  for (const winner of playerWinners) {
+    PLAYER_WIN_COUNT[winner] += 1;
+  }
+
+  return playerWinners;
+};
+
+/* Helpers to get display */
+const getDisplayPrevRoundResult = function () {
+  return PREV_ROUND_RESULT_DESCRIPTION;
+};
+
+const getDisplayOrdering = function (ordering) {
+  var line = "";
+  for (var j = 0; j < ordering.length; j++) {
+    line += ordering[j];
+  }
+  return line;
+};
+
+const getDisplayOrderingsOfPlayer = function (playerIndex) {
+  var orderings = playersRollsOrderings[playerIndex];
+  var lines = "";
+  for (let i = 0; i < orderings.length; i += 1) {
+    var line = `[${i}]: `;
+    var ordering = orderings[i];
+    line += getDisplayOrdering(ordering);
+    lines += line + "<br />";
+  }
+  return lines;
+};
+
+const displayCurrentPlayer = function (PLAYER) {
+  if (PLAYER_ONE === PLAYER) {
+    return "PLAYER 1";
+  } else if (PLAYER_TWO === PLAYER) {
+    return "PLAYER 2";
+  } else {
+    console.log(PLAYER + "not " + PLAYER_ONE + PLAYER_TWO);
+  }
+};
+
+/**
+ *          Player 1    Player 2
+ *  Roll     3            2
+ *  Choice    1           1
+ */
+
+const generateRoundStatisticsTable = function () {
+  var table = document.createElement("table");
+  table.className += ` flex-table`;
+  var headerRow = table.insertRow();
+  headerRow.insertCell().innerHTML = "#(Total Wins W): ";
+  for (let playerIndex = 0; playerIndex < PLAYERS; playerIndex++) {
+    headerRow.insertCell().innerHTML = `${displayCurrentPlayer(
+      playerIndex
+    )}  (${PLAYER_WIN_COUNT[playerIndex]})`;
+  }
+
+  var rollRow = table.insertRow();
+  rollRow.insertCell().innerHTML = "Roll: ";
+  for (let playerIndex = 0; playerIndex < PLAYERS; playerIndex++) {
+    rollRow.insertCell().innerHTML = playerRolls[playerIndex];
+  }
+
+  var choiceOfOrderingRow = table.insertRow();
+  choiceOfOrderingRow.insertCell().innerHTML = "Choice: ";
+  for (let playerIndex = 0; playerIndex < PLAYERS; playerIndex++) {
+    choiceOfOrderingRow.insertCell().innerHTML =
+      playerOrderingChoice[playerIndex];
+  }
+
+  return table;
+};
+
+/* Helpers to set innerHTML */
+const setInputDescription = function (text) {
+  HTML_G_INPUT_DESC.innerHTML = text;
+};
+const setButtonValue = function (text) {
+  HTML_G_BUTTON.innerHTML = text;
+};
```

Comment:
again, same comment as above for helper functions with simple one-line assignments. in this case i understand it more because you prob wish to abstract away the DOM manipulation from the core function, but i think the fact that you already your DOM element stored in variables makes it looks clean enough as is